From: Sorah Fukumori <her@sorah.jp>
Date: Tue, 9 Feb 2021 13:02:14 +0900
Subject: include mitamae-secrets

---
 build_config.rb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/build_config.rb b/build_config.rb
index d5e7ea9..30308c9 100644
--- a/build_config.rb
+++ b/build_config.rb
@@ -1,5 +1,7 @@
 def gem_config(conf)
   conf.gem File.expand_path(File.dirname(__FILE__))
+
+  conf.gem github: 'eagletmt/mitamae-secrets', checksum_hash: 'v0.1.1'
 end
 
 def debug_config(conf)
